contract/build.gradle

```diff
@@ -0,0 +1,116 @@
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+
+plugins {
+    id "java-library"
+    id 'org.openapi.generator' version '7.9.0'
```

Comment:
can we extract this version as well to a version catalog (gradle's "dependency management" list) to make it possible for dependabot to track these versions as well?

Reply:
Ready